src/main/java/edu/hawaii/its/api/controller/ErrorControllerAdvice.java

```diff
@@ -27,63 +31,165 @@ public class ErrorControllerAdvice {
     @Autowired
     private EmailService emailService;
 
+    private ResponseEntity<Object> buildResponseEntity(ApiError apiError) {
+        return new ResponseEntity<>(apiError, apiError.getStatus());
+    }
+
     @ExceptionHandler(GroupingsServiceResultException.class)
-    public ResponseEntity<GroupingsHTTPException> handleGroupingsServiceResultException(
+    public ResponseEntity<Object> handleGroupingsServiceResultException(
```

Comment:
If possible, could you set the return type to "ReponseEntity<ApiError>" for all of the functions in here? Thanks.

Reply:
Sorry, I formatted my message wrong and made `<ApiError>` invisible.